style/input.less

```diff
@@ -0,0 +1,10 @@
+* {
```

Comment:
I think it would be fine to write this a bit more compact since the intermediate levels don't have any style.

```
*[dir=rtl] {
  input[nav-selected=true] {
    // style here
  }
}
```